modules/efk-logging-kibana-limits.adoc

```diff
@@ -5,18 +5,20 @@
 [id="efk-logging-kibana-limits-{context}"]
 = Configure Kibana CPU and memory limits
 
-Each component specification allows for adjustments to both the CPU and memory limits. 
+Each component specification allows for adjustments to both the CPU and memory limits. You should not need to manually adjust these values as the Cluster Logging Operator sets values sufficient for your environment.
 
 .Prerequisite
 
-If needed, get the name of the Cluster Logging Custom Resource from the `openshift-logging` project:
-
+* If needed, get the name of the Cluster Logging Custom Resource from the `openshift-logging` project:
++
 ----
 $ oc get ClusterLogging
 NAME       AGE
 instance   112m
 ----
 
+* Set cluster logging to the unmanaged state. 
```

Comment:
This is not needed in order to adjust the proxy settings.

Reply:
@jcantrill What about for memory and CPU requests for Kibana (not the proxy)?

Reply:
@mburke5678  The change is in Cluster Logging Custom Resource, the cluster logging must be  in Managed state. 

The other Part LGTM